atomix/cluster/src/test/resources/log4j2-test.xml

```diff
@@ -9,7 +9,7 @@
 
   <Loggers>
     <Logger name="io.zeebe" level="debug"/>
-    <Logger name="io.atomix" level="debug"/>
+    <Logger name="io.atomix" level="trace"/>
```

Comment:
hm maybe we shouldn't put this to trace :sweat_smile:

Reply:
If trace was enabled, debugging would have been easier by reading the logs from jenkins.  Traces gives an idea on the order of these operations. Is it too much to enable traces for test? It's not production.

Reply:
>  It's not production.

True. But it still is quite noisy even debugging level and increases the log sizes of all tests, which is not necessary. But if you disagree just leave it as it is.

Reply:
For some tests it could be noisy. But as long as no important log is removed, it would be ok right? In most cases I cannot reproduce these flaky tests locally and I rely on the jenkins logs to debug. If there are no relevant logs, then I have to spend a lot of time thinking what could have gone wrong, like in this case. How about enabling it trace now, and if it creates problems for other tests turn it back to debug?

Reply:
sounds good :+1: